src/api/app/assets/javascripts/webui/application.js

```diff
@@ -246,3 +246,13 @@ function select_from_autocomplete(toselect) {
         if ($(this).text() == toselect) { $(this).trigger('mouseenter').click(); }
     });
 }
+
+$(function() {
+  $('.show_dialog').on('click', function() {
+    $($(this).data('target')).removeClass('hidden');
+  });
+
+  $('.close-dialog').on('click', function() {
```

Comment:
beginner's error: you can't bind elements that aren't there. I suggest you stop merging refactorings without test :(